specification/resources/resource-manager/Microsoft.Resources/stable/2022-12-01/subscriptions.json

```diff
@@ -0,0 +1,874 @@
+{
```

Comment:
Not sure why your PR shows so many Linter Errors. The only changes in this PR are: adding AvailabilityZoneMappings and geography property, right?

Reply:
yes. That's correct